packages/accordion/src/AccordionContainer.spec.tsx

```diff
@@ -9,43 +9,43 @@ import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { KEY_CODES } from '@zendeskgarden/container-utilities';
 import { AccordionContainer } from './AccordionContainer';
+import { IUseAccordionReturnValue } from './useAccordion';
 
 describe('AccordionContainer', () => {
   it('renders with expected return values', () => {
-    const { getByTestId } = render(
+    const { getByTestId, getByText } = render(
       <AccordionContainer>
-        {({ expandedSections, disabledSections }) => (
-          <div
-            data-test-id="test"
-            hidden={expandedSections.length === 0}
-            disabled={disabledSections.length !== 0}
-          />
+        {({ expandedSections, disabledSections }: IUseAccordionReturnValue) => (
+          <div data-test-id="test" hidden={expandedSections.length === 0}>
+            <button disabled={disabledSections.length !== 0}>Trigger</button>
+          </div>
         )}
       </AccordionContainer>
     );
     const test = getByTestId('test');
+    const triggerButton = getByText('Trigger');
 
     expect(test).toHaveAttribute('hidden');
-    expect(test).not.toHaveAttribute('disabled');
+    expect(triggerButton).not.toHaveAttribute('disabled');
   });
 
-  const sections = Array(3).fill();
+  const sections = Array(3).fill(undefined);
```

Comment:
A TypeScript warning says `.fill` must take at least a single argument. By passing `undefined`, the result is the same as calling `.fill` with no arguments: an array filled with `undefined` values.